src/integration-test/java/com/mongodb/jdbc/integration/testharness/models/TestEntry.java

```diff
@@ -13,6 +13,7 @@
     public Boolean ordered;
     public List<Object> expected_result;
     public List<String> expected_sql_type;
+    public List<String> expected_bson_type;
```

Comment:
should we generate this row for all the already committed integration tests? Note that several of them have been manually edited so we shouldn't replace them wholesale. We should just generate and then add _this field_ to the tests.

Reply:
It may also be fine to _not_ add this field to the existing tests since they don't specifically contain types that would lead to interesting results here.

Reply:
I added the additional `expected_bson_type` field.  There were `string`, `int`, `bool`, `long` types.